docs/website/docs/dlt-ecosystem/destinations/athena.md

```diff
@@ -0,0 +1,19 @@
+---
+title: AWS Athena / Glue Catalog
+description: AWS Athena `dlt` destination
+keywords: [aws, athena, glue catalog]
+---
+
+# AWS Athena
+
+### Setup:
+* Provide aws credentials for an identity that has the athena full access role
+* Provide an s3 bucket which serves as the output for Athena queries, the aws identity will need access to this bucket too.
+* Ensure that the aws identity also had read and write access for the buckets that will have the files for the athena tables
```

Comment:
maybe providing a basic set of iam permissions needed to make the setup working will be better.